gifs/src/main/kotlin/jetbrains/kotlin/course/gifs/almost/done/configs/GetPicture.yaml

```diff
@@ -0,0 +1,29 @@
+terminal_config:
+  cols: 70
+  rows: 8
+
+interaction_config:
+  action_delay: 200
+  keystroke_delay: 75
+
+gif_config:
+  common:
+    font_size: 32
+    font_family: JetBrains Mono
+
+  gifs:
+    - name: 'get_picture_dark'
+      theme: 'jetbrains_dark'
+
+    - name: 'get_picture'
+      theme: 'jetbrains_light'
+
+scenario:
+  - expect: \n
```

Comment:
Does it work on Windows?

Reply:
Anderson doesn't work on Windows now due to technical limitations 😢

Reply:
oh I totally forgot, ok